### ISSUE TITLE
BSD buffer sizes listed on https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes#bsd  missing trailing '0'

### ISSUE THREAD

**Issue by jimscard** [quic-go/quic-go#4351]
The buffer sizes in the example command lines at the bottom of https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes#bsd are missing the final zero, so they are 10 times too small.

**Comment by marten-seemann** [comment]
I don't see it. The number seems to be around 3 million, corresponding to a buffer size of ~2.5 MB, doesn't it?

**Comment by jimscard** [comment]
Oh yeah, now that you mention it, there's more wrong on the page than I thought. You're right, it does correspond to about 2.5MB plus a 15% roundup. However, that's 10 times too small. It should be 25MB plus the round up. 

If you follow the link on the page to the Cameron Sparr post, you'll see it says "It is highly recommended that you increase these OS limits to at least 25MB before trying to run UDP traffic to your server. 25MB is just a recommendation." 

The non-BSD values on your page are a factor of too small as well. Sparr's recommendation for them is 26214400.

I noticed this when I copy & pasted the values from the quic-go page to my macbook and got a response that showed that the buffer size had gotten *smaller*.  Turns out the default is  `kern.ipc.maxsockbuf: 8388608`
